### PR TITLE
🌱 clusterctl: fix typo for GitHub rate limit error

### PR DESCRIPTION
**Pull request by damdo**
**What this PR does / why we need it**:
This PR fixes a typo in the GitHub rate limit text.

### CODE REVIEW

File: cmd/clusterctl/client/cluster/template.go [hunk 1]
```diff
@@ -215,7 +215,7 @@ func getGitHubClient(configVariablesClient config.VariablesClient) (*github.Clie
 // handleGithubErr wraps error messages.
 func handleGithubErr(err error, message string, args ...interface{}) error {
 	if _, ok := err.(*github.RateLimitError); ok {
-		return errors.New("rate limit for github api has been reached. Please wait one hour or get a personal API tokens a assign it to the GITHUB_TOKEN environment variable")
+		return errors.New("rate limit for github api has been reached. Please wait one hour or get a personal API tokens and assign it to the GITHUB_TOKEN environment variable")
```
Comment (sbueringer) [inline cmd/clusterctl/client/cluster/template.go:218]:
```suggestion
		return errors.New("rate limit for github api has been reached. Please wait one hour or get a personal API token and assign it to the GITHUB_TOKEN environment variable")
```
nit, same below

Comment (damdo) [reply cmd/clusterctl/client/cluster/template.go]: Thanks, included.

**Comment by damdo** [comment]
Included some new changes caught by @sbueringer. We'll need the `lgtm`s again, @stmcginnis @killianmuldoon, thanks.

**Comment by stmcginnis** [comment]
Good call...

/lgtm

**Comment by sbueringer** [comment]
Thx!

@damdo Can you please squash the commits? (we're running Prow without squash to preserve information for our release note generation tool)

 Otherwise lgtm

**Comment by damdo** [comment]
Done 👍 thanks for the guidance @sbueringer.

**Comment by sbueringer** [comment]
You're welcome :)

/lgtm

**Review by fabriziopandini — COMMENTED** [review]
@damdo congrats on your first PR on CAPI! Welcome to the team!
/lgtm
/approve

**Pull request merged**